osu.Game/Overlays/BeatmapSet/Scores/ScoresContainer.cs

```diff
@@ -223,6 +235,18 @@ private void getScores()
                 return;
             }
 
+            if (scope.Value != BeatmapLeaderboardScope.Global && !userIsSupporter)
+            {
+                Scores = null;
+                notSupporterPlaceholder.Show();
+                loading.Hide();
+                return;
+            }
+            else
+            {
+                notSupporterPlaceholder.Hide();
+            }
```

Comment:
If you're going to `return` in the first branch then there's no need to have an `else` at all.